src/cloud-api-adaptor/test/e2e/assessment_helpers.go

```diff
@@ -58,6 +58,10 @@ func NewTestCase(t *testing.T, e env.Environment, testName string, assert CloudA
 		deletionWithin: assert.DefaultTimeout(),
 	}
 
+	if os.Getenv("TEST_SECURE_COMMS") == "true" {
+		testCase.WithSecureComms()
+	}
```

Comment:
You already have `TEST_E2E_SECURE_COMMS` set, so can't you use the value of this rather than setting another env?

Reply:
This mechanism was replaced